hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -637,6 +637,48 @@ objects:
           \      operator: Exists\nkubeStateMetrics:\n  nodeSelector:\n    node-role.kubernetes.io/infra:\
           \ \"\"\n  tolerations: \n    - effect: NoSchedule\n      key: node-role.kubernetes.io/infra\n\
           \      operator: Exists"
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: cluster-monitoring-config-uwm-canary
```

Comment:
This will fight the other SelectSyncSet `cluster-monitoring-config` we need to exclude the other from being applied to clusters with the label `api.openshift.com/techpreview/uwm: "true"`

Reply:
i wonder if we should just set `api.openshift.com/techpreview/uwm: "false"` by default on all clusterdeployments and add that to the default SSS for monitoring